app/plugins/prometheus.md

```diff
@@ -0,0 +1,171 @@
+# TYPE kong_latency histogram
+kong_latency_bucket{type="kong",service="google",le="00001.0"} 1
+kong_latency_bucket{type="kong",service="google",le="00002.0"} 1
+.
+.
+.
+kong_latency_bucket{type="kong",service="google",le="+Inf"} 2
+kong_latency_bucket{type="request",service="google",le="00300.0"} 1
+kong_latency_bucket{type="request",service="google",le="00400.0"} 1
+.
+.
+kong_latency_bucket{type="request",service="google",le="+Inf"} 2
+kong_latency_bucket{type="upstream",service="google",le="00300.0"} 2
+kong_latency_bucket{type="upstream",service="google",le="00400.0"} 2
+.
+.
+kong_latency_bucket{type="upstream",service="google",le="+Inf"} 2
+kong_latency_count{type="kong",service="google"} 2
+kong_latency_count{type="request",service="google"} 2
+kong_latency_count{type="upstream",service="google"} 2
+kong_latency_sum{type="kong",service="google"} 2145
+kong_latency_sum{type="request",service="google"} 2672
+kong_latency_sum{type="upstream",service="google"} 527
+# HELP kong_latency_total Latency added by Kong, total request time and upstream latency aggreggated across all services in Kong
+# TYPE kong_latency_total histogram
+kong_latency_total_bucket{type="kong",le="00001.0"} 1
+kong_latency_total_bucket{type="kong",le="00002.0"} 1
+.
+.
+kong_latency_total_bucket{type="kong",le="+Inf"} 2
+kong_latency_total_bucket{type="request",le="00300.0"} 1
+kong_latency_total_bucket{type="request",le="00400.0"} 1
+.
+.
+kong_latency_total_bucket{type="request",le="+Inf"} 2
+kong_latency_total_bucket{type="upstream",le="00300.0"} 2
+kong_latency_total_bucket{type="upstream",le="00400.0"} 2
+.
+.
+.
+kong_latency_total_bucket{type="upstream",le="+Inf"} 2
+kong_latency_total_count{type="kong"} 2
+kong_latency_total_count{type="request"} 2
+kong_latency_total_count{type="upstream"} 2
+kong_latency_total_sum{type="kong"} 2145
+kong_latency_total_sum{type="request"} 2672
+kong_latency_total_sum{type="upstream"} 527
+# HELP kong_nginx_http_current_connections Number of HTTP connections
+# TYPE kong_nginx_http_current_connections gauge
+kong_nginx_http_current_connections{state="accepted"} 8
+kong_nginx_http_current_connections{state="active"} 1
+kong_nginx_http_current_connections{state="handled"} 8
+kong_nginx_http_current_connections{state="reading"} 0
+kong_nginx_http_current_connections{state="total"} 8
+kong_nginx_http_current_connections{state="waiting"} 0
+kong_nginx_http_current_connections{state="writing"} 1
+# HELP kong_nginx_metric_errors_total Number of nginx-lua-prometheus errors
+# TYPE kong_nginx_metric_errors_total counter
+kong_nginx_metric_errors_total 0
+
+```
+
+In most configurations the Kong Admin API will be behind a firewall or would
+need to be setup to require authentication, here are a couple of options to
+allow access to the `/metrics` endpoint to Prometheus.
+
+
+1. Kong Enterprise users can protect the admin `/metrics` endpoint with an
```

Comment:
Since this is specific to Kong Enterprise and RBAC, can we move this to the end of this list?

Reply:
I strongly disagree with moving this to the bottom. It is important that we use opportunities like these to let people know that there is an Enterprise version of Kong, and that by referencing it first, show that it is important to us and our customers.